### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by szepeviktor**
@tangrufus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # PHP Matrix
 
 [![Packagist Version](https://img.shields.io/packagist/v/typisttech/php-matrix)](https://packagist.org/packages/typisttech/php-matrix)
-[![PHP Version Require](https://img.shields.io/packagist/dependency-v/typisttech/php-matrix/php)]([https://github.com/typisttech/php-matrix/blob/readme/composer.json](https://github.com/typisttech/php-matrix/blob/main/composer.json))
+[![PHP Version Require](https://img.shields.io/packagist/dependency-v/typisttech/php-matrix/php)](https://github.com/typisttech/php-matrix/blob/main/composer.json)
```
Comment (szepeviktor) [inline README.md:6]: Fix the badge.

File: README.md [hunk 2]
```diff
@@ -14,8 +14,8 @@
 
 <p>
   <strong>List PHP versions that satisfy the given constraint.</strong>
-  <br />
-  <br />
```
Comment (szepeviktor) [inline README.md:18]: no more HTML4

File: README.md [hunk 3]
```diff
@@ -115,9 +115,9 @@ cd php-matrix
 ## Credits
 
 [`PHP Matrix`](https://github.com/typisttech/php-matrix) is a [Typist Tech](https://typist.tech) project and
-maintained by [Tang Rufus](https://x.com/TangRufus), freelance developer for [hire](https://typist.tech/contact/).
+maintained by [Tang Rufus](https://x.com/TangRufus), freelance developer [for hire](https://typist.tech/contact/).
```
Comment (szepeviktor) [inline README.md:118]: "for hire" has the meaning

File: README.md [hunk 4]
```diff
@@ -115,9 +115,9 @@ cd php-matrix
...
 
-Full list of contributors can be found [here](https://github.com/typisttech/php-matrix/graphs/contributors).
+Full list of contributors can be found [on GitHub](https://github.com/typisttech/php-matrix/graphs/contributors).
```
Comment (szepeviktor) [inline README.md:120]: Never link the word "here".

File: README.md (hunk 1 above)
Comment (szepeviktor) [reply README.md:6]: When pasting an URL GitHub creates a link ...

**Comment by tangrufus** [comment]
Much better!

Thank you 👍

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**